pkg/plugin/system/system_test.go

```diff
@@ -279,3 +284,303 @@ func TestPluginSystemInterceptors(t *testing.T) {
 		g.Expect(err.(*aspecterrors.ExitError).ExitCode).To(Equal(1))
 	})
 }
+
+func TestConfigure(t *testing.T) {
+	t.Run("fails when Finder fails to find plugin config file", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+
+		var stdout strings.Builder
+		streams := ioutils.Streams{Stdout: &stdout, Stderr: &stdout}
+
+		finder := loader_mock.NewMockFinder(ctrl)
+		finder.EXPECT().Find().Return("", errors.New("finderr"))
+
+		ps := &pluginSystem{finder: finder}
+
+		err := ps.Configure(streams)
+
+		g.Expect(err).NotTo(BeNil())
```

Comment:
Gotta check if the error is the same as you expect. Not looking at the implementation of `Configure`, how can we be sure the error is the one from `Find`? The same applies below.